# Vector of MISSION_ITEM, see MAVlink
uint16 mission_item_number
uint16[100] command
float32[100] param1
float32[100] param2
float32[100] param3
float32[100] param4
float32[100] param5
float32[100] param6
float32[100] param7